wagtail/tests/testapp/models.py

```diff
@@ -1007,12 +1007,38 @@ class StreamModel(models.Model):
     ])
 
 
+class MinMaxCountStreamModel(models.Model):
+    body = StreamField([
+        ('text', CharBlock()),
+        ('rich_text', RichTextBlock()),
+        ('image', ImageChooserBlock()),
+    ],
+        min_num=2,
+        max_num=5,
+    )
+
+
+class BlockCountsStreamModel(models.Model):
+    body = StreamField([
+        ('text', CharBlock()),
+        ('rich_text', RichTextBlock()),
+        ('image', ImageChooserBlock()),
+    ],
+        block_counts={
```

Comment:
There's a couple of indentation issues here, surprised that flake8 didn't catch this!